ui/nuxeo-document-comments/nuxeo-document-comments-mixin.js

```diff
@@ -0,0 +1,70 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+
+const CommentsMixin = (superClass) =>
```

Comment:
Mixed feelings too... In terms of code, it is not worse than it was, since we remove duplication and we centralize the comments input area. Anyway, it is not an element by itself neither a behaviour.

Reply:
**Update: Removed**
Let's keep as it, for now, as was in WebUI.